pubsublite/ps/example_test.go

```diff
@@ -72,3 +80,61 @@ func ExamplePublisherClient_Error() {
 		fmt.Printf("Published a message with a message ID: %s\n", id)
 	}
 }
+
+func ExampleSubscriberClient_Receive() {
+	ctx := context.Background()
+	subscription := pubsublite.SubscriptionPath{
+		Project:        "project-id",
+		Zone:           "zone",
+		SubscriptionID: "subscription-id",
+	}
+	subscriber, err := ps.NewSubscriberClient(ctx, ps.DefaultReceiveSettings, subscription)
```

Comment:
Perhaps we should clarify that we don't need to pass in `ps.DefaultReceiveSettings` to achieve default functionality

Reply:
Good point. Added a note "DefaultReceiveSettings and empty ReceiveSettings{} are equivalent" and same for the PublishSettings example above.